apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -1829,15 +1866,15 @@ input:-moz-placeholder {
     font-size: 12px;
 }
 
-.company-name ul li>span {
+.company-name ul li > span {
     color: #333 !important;
     text-overflow: ellipsis;
     white-space: nowrap;
     overflow: hidden;
     font-size: 14px;
 }
 
-.company-name .cText .cName {
+.company-name .c-text .c-name {
```

Comment:
Add generic class name

Reply:
generic class name will be covered in code designing